src/app/views/sub-tenant/update-account-managers-modal/update-account-managers.controller.coffee

```diff
@@ -78,4 +76,31 @@
   vm.onCancel = () ->
     $uibModalInstance.dismiss('cancel')
 
+  vm.isDisable = () ->
+    vm.isLoading || (vm.changes.add.length == 0 && vm.changes.remove.length == 0)
+
+  vm.checkBoxChanged = (user) ->
+    if user.belong_to_sub_tenant
+      index = _.indexOf(vm.changes.remove, user.id)
+      if index == -1
```

Comment:
Use `_.contains()` for more code clarity. Also, it stops looping at the first occurrence.

Reply:
Yeah, but I need the index to remove the element.

Reply:
👍